cobalt/BUILD.gn

```diff
@@ -13,6 +13,44 @@
 # limitations under the License.
 
 group("gn_all") {
+  testonly = true
```

Comment:
GN `testonly` is reserved for targets that produce test files.
So this should be removed. Same with l.27. See

https://source.chromium.org/search?q=file:BUILD.gn%20testonly%20file:(content%7Cbase%7Cmedia)&ss=chromium%2Fchromium%2Fsrc

for some uses.

Reply:
If i remove it, it won't build because the top level gn_all has it.